.github/workflows/shared-release-pull-request-library-java.yml

```diff
@@ -169,32 +157,9 @@ jobs:
           body: ${{ steps.release-notes-markdown.outputs.releaseNotes }}
           tag: ${{ needs.determine-version.outputs.version }}
 
-      - name: Create JIRA Release
```

Comment:
No need to create Jira release for libraries

Reply:
No need to, but if the library build is the product, isn't that still a release?

Reply:
The change should be captured by the release for the service importing the library.

Reply:
I don't want to get too philosophical here but essentially I disagree
As long as the library is a standalone thing with its own versioning and artefact (and repository), any change to it is a change to its capability or perhaps a bug fix to its capability
There was a new release version 1.5.3 of the common-base library recently.  It should really have been 1.6.0 because it introduced support for a ForbiddenException
But that was a new release of that product.  There may have been a jira subtask created for its dependency, but it literally was a new release of a new version of that library with new capability
I don't know what service is going to use it, but it doesn't matter whether 0, 1 or many services do, we have a new release of common-base and the change is captured by the release of the library, not by the release of anything that may happen to use it

Reply:
A library isn't really a product on its own. Releases for libraries are already tracked under the GitHub releases https://github.com/cartoncloud/common-service-starter/releases.

Jira release are intended to be production releases of a product. The idea was to get rid of the extra noise it adds in Jira as when libraries are marked release, there is no change (at that time) to the product to the end user.
<img width="608" alt="image" src="https://github.com/cartoncloud/actions/assets/1125196/0e53d84a-77e3-4c14-a121-ee37fb572d04">